user_tools/src/spark_rapids_pytools/rapids/rapids_tool.py

```diff
@@ -307,7 +322,24 @@ def _process_tool_args(self):
                 else:
                     # this could be a boolean type flag that has no arguments
                     arguments_list.append(f'{k_arg}')
-        self.ctxt.add_rapids_args('rapidsOpts', arguments_list)
+        return arguments_list
+
+    def _append_tool_rapids_args(self, cli_rapids_options: list) -> list:
+        """
+        Specific to the tool, add the rapids arguments needed to run the plugin core tools
+        :param cli_rapids_options:
+        :return:
+        """
+        return cli_rapids_options
```

Comment:
This function only returns its argument. Is this the desired behavior?

Reply:
Yes, it was intentional because it was overrriden in the profiling tool.
However, after reviewing I found that we can get rid of this method. So, I removed it in the latest commit.